### DISCUSSION TITLE
手动设置global规则问题

### DISCUSSION THREAD

**Discussion by hussion** [kkkgo/PaoPaoGateWay#54] [问和答]
遇到一个奇怪的问题：
1. 通过suburl配置的URL订阅方式，通过`Rule`规则，科学上网一切正常，手动切换为`Global`方式，TG的IP连接都超时，导致无法正常使用tg
2. 顺便问下，如何清除fake-ip的缓存呢？能否提供一个快捷方式？

**Comment by kkkgo** [comment]
使用Global模块你要自己选好节点。或者你直接开启fast_node=yes，就是Global模式自动选节点。
不需要清除fakeip缓存，他在运行过程中是持久化的。网关没有硬盘，直接重启就是你要的清除。
如果是担心fakeip缓存问题，建议手动docker定制镜像SNIFF=yes。

**Reply by hussion** [reply]
节点已经选好了的，来来回回试了好几次都不行

**Reply by kkkgo** [reply]
你选的节点可能是规则里面的代理组，或者你的代理本身需要代理链等。总之，你可以点debug日志来观察连接到了什么节点，或者连接失败的信息，或者直接尝试使用fast_node=yes。

**Reply by hussion** [reply]
我现在选的节点确实是代理组里的某个节点，正确的应该怎么做？`fast_node=yes`是可以的，但是我的梯子不稳定，经常会导致选择的节点不可用，如果不用`fast_node=yes`，如何正确的手动设置Global模式呢？

**Reply by kkkgo** [reply]
`fast_node=yes`当节点不可用的时候会自动切换最快的可用节点。`fast_node=yes`不影响你手动选择节点，因为他仅在节点不可用的时候会切走你选的节点。

**Reply by kkkgo** [reply]
你应该选择真正的出口节点而不是代理组，代理组可能是走规则的。`fast_node=yes`能用只是防呆忽略加载所有规则。

**Reply by hussion** [reply]
具体该怎么选择呢？是在Global那个组里选节点吗？
![image](https://github.com/kkkgo/PaoPaoGateWay/assets/1524545/d0d9475c-ea5d-47d8-8aea-f9d6a87ffa44)

**Reply by kkkgo** [reply]
正常来说是的，你也可以通过转换订阅或者手动编辑把代理组和规则去掉。

**Comment by hussion** [comment]
> `fast_node=yes`当节点不可用的时候会自动切换最快的可用节点。`fast_node=yes`不影响你手动选择节点，因为他仅在节点不可用的时候会切走你选的节点。

一开始用的也是`fast_node=yes`，但是实际情况自动帮我选的节点总是不可用，即使我已经手动选择了节点，没办法了才把fast_node关闭了

**Reply by kkkgo** [reply]
fast_node测试的是test_node_url参数的可用性，你可以通过改变test_node_url的值来达到你的测试可用的需求。
